unison-core/src/Unison/HashQualified'.hs

```diff
@@ -17,110 +18,108 @@ import           Unison.ShortHash               ( ShortHash )
 import qualified Unison.ShortHash              as SH
 import qualified Unison.HashQualified          as HQ
 
-data HashQualified' n = NameOnly n | HashQualified n ShortHash
-  deriving (Eq, Functor)
+data HashQualified n = NameOnly n | HashQualified n ShortHash
```

Comment:
The changes to `HashQualified` here are just to make the code a little easier to read. The `HashQualified` type was just an alias for `HashQualified' Name`, so I've removed the `'` from that type and removed the alias.

In particular I wanted to use `HashQualified NameSegment` in some places, and I had a hard time keeping a handle on what all the `'` were for.

Reply:
I don't think codebase2 deals with `HashQualified` at all (though `Referent` is a different story), but I do feel like I'm absolutely drowning in renames for this PR, and concerned that I don't really have attention cycles remaining for any of the interesting codebase server stuff.

Reply:
Needed to remove the `'` here to satisfy Servant. The alternative is to duplicate this type into the Server module.